src/core/probe/bpf/include/common_defs.h

```diff
@@ -1,7 +1,56 @@
 #ifndef __CORE_PROBE_COMMON_DEFS__
 #define __CORE_PROBE_COMMON_DEFS__
 
+#include <vmlinux.h>
+
+#include "events.h"
+
+/* Keep in sync with its Rust counterpart in crate::core::probe */
+#define PROBE_MAX	1024
+
 #define COMMON_SECTION_CORE	0
 #define COMMON_SECTION_TASK	1
 
+/* Error reporting is enabled. */
+const volatile u8 report_error = 0;
+
+struct retis_counters_key {
+	/* Symbol address. */
+	u64 sym_addr;
+	/* pid of the process. Zero is used for the
+	 * kernel as it is normally reserved the swapper task. */
+	u64 pid;
+};
+
+/* Contains the counters of the error path.  This is then processed
+ * and reported from user-space. */
+struct retis_counters {
+	u64 events;
+};
+
+/* Probe configuration; the key is the target symbol address */
+struct {
+	__uint(type, BPF_MAP_TYPE_HASH);
+	__uint(max_entries, PROBE_MAX * 2);
```

Comment:
PROBE_MAX was thought for kernel probes.
I initially doubled it because it still was a kernel entity. We should probably give it a fixed value without doubling it and use a different value for the kernel max number of probes. 
This map should take into account KERNEL_PROBES_MAX + USDT_PROBES_MAX although I expect USDTs to remain fairly contained in terms of numbers
WDYT?

Reply:
> > PROBE_MAX was thought for kernel probes.
> 
> No this is actually the global limit, see `check_probe_max` in the probe manager. The kernel config map do use it as it's the upper limit, but it's not kernel specific.

ah, right. My memory failed. I was sure it was defined under the kernel probe part (which seemed a bit weird, btw), but that simply was not true. So, '*2' doesn't make any sense and just need to be dropped